portfolio/src/main/webapp/script.js

```diff
@@ -140,6 +142,16 @@ function showModal() {
     modalBody.appendChild(mapElement);
     modalBody.appendChild(mapForm); 
   }
+  
+    activityButton.onclick = function () {
+    modalBody.innerHTML= '';
+    modal.style.display = 'block';
+    modalImage.style.display = 'none';
+    modalHeader.innerText = 'Weekend Activities';
+    chartElement.style.display = 'block';
+    drawChart(chartElement);  
+    modalBody.appendChild(chartElement);
+  }
```

Comment:
lines 147-154 need to be indented one more

Reply:
Whoops, messed up the indent for line 146